examples/make_eeg_bids.py

```diff
@@ -0,0 +1,199 @@
+Use MNE-BIDS for EEG Data
+=========================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+#. Download some EEG data from the
+`PhysioBank database <https://physionet.org/physiobank/database>`_.
+
+#. Load the data, extract information, and save in a new BIDS directory
+
+#. Check the result and compare it with the standard
+
+"""
+
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+# License: BSD (3-clause)
+
+###############################################################################
+# We are importing everything we need for this example:
+import os
+import shutil as sh
+
+from mne.datasets import eegbci
+from mne.io import read_raw_edf
+
+from mne_bids import raw_to_bids
+from mne_bids.utils import print_dir_tree
+
+###############################################################################
+# Step 1: Download the data
+# -------------------------
+#
+# First, we need some data to work with. We will use the
+# `EEG Motor Movement/Imagery Dataset <https://doi.org/10.13026/C28G6P>`_
+# available on the PhysioBank database.
+#
+# The data consists of 109 volunteers performing 14 experimental runs each. For
+# each subject, there were two baseline tasks (1. eyes open, 2. eyes closed) as
+# well as four different motor imagery tasks. For the present example, we will
+# show how to format the data for two subjects and selected tasks to comply
+# with the Brain Imaging Data Structure
+# (`BIDS <http://bids.neuroimaging.io/>`).
+#
+# Conveniently, there is already a data loading function available with
+# MNE-Python:
+
+# Make a directory to save the data to
+home = os.path.expanduser('~')
+mne_dir = os.path.join(home, 'mne_data')
+if not os.path.exists(mne_dir):
+    os.makedirs(mne_dir)
+
+# Define which tasks we want to download.
+tasks = [2,  # This is 2 minutes of eyes closed rest
+         4,  # This is run #1 of imagining to close left or right fist
+         12]  # This is run #2 of imagining to close left or right fist
+
+# Download the data for subjects 1 and 2
+for subj_idx in [1, 2]:
+    eegbci.load_data(subject=subj_idx,
+                     runs=tasks,
+                     path=mne_dir,
+                     update_path=True)
+
+###############################################################################
+# Let's see whether the data has been downloaded using a quick visualization
+# of the directory tree.
+
+data_dir = os.path.join(mne_dir, 'MNE-eegbci-data')
+print_dir_tree(data_dir)
+
+###############################################################################
+# The data are in the `European Data Format <https://www.edfplus.info/>`_
+# '.edf', which is good for us because next to the BrainVision format, EDF is
+# one of the recommended file formats for EEG BIDS. However, apart from the
+# data format, we need to build a directory structure and supply meta data
+# files to properly *bidsify* this data.
```

Comment:
bidsify in italics maybe? :)

Reply:
this is already in italics: 
![image](https://user-images.githubusercontent.com/9084751/45535123-b10f9f00-b7fd-11e8-9230-f792a9f983a0.png)